src/syntax/rules.rkt

```diff
@@ -4,7 +4,7 @@
 
 (require "../common.rkt")
 
-(provide (struct-out rule) *rules* *simplify-rules* *fp-safe-simplify-rules*)
+(provide (struct-out rule) *complex-rules* rule-valid-at-type? *rules* *simplify-rules* *fp-safe-simplify-rules*)
```

Comment:
I think that David's changes (adding fp-safe rules) and your changes merged poorly—there are some groups that seem like they should be good for complex numbers but aren't marked `complex`. It'd be worth looking the rules over again to check.